src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_ConfirmProviderHiddenPartial.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort.ConfirmProviderViewModel
+
+
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.AccountHashedId)" name="@nameof(ConfirmProviderViewModel.AccountHashedId)" value="@Model.AccountHashedId"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.EmployerAccountLegalEntityPublicHashedId)" name="@nameof(ConfirmProviderViewModel.EmployerAccountLegalEntityPublicHashedId)" value="@Model.EmployerAccountLegalEntityPublicHashedId"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.ReservationId)" name="@nameof(ConfirmProviderViewModel.ReservationId)" value="@Model.ReservationId"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.CourseCode)" name="@nameof(ConfirmProviderViewModel.CourseCode)" value="@Model.CourseCode"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.StartMonthYear)" name="@nameof(ConfirmProviderViewModel.StartMonthYear)" value="@Model.StartMonthYear"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.ProviderId)" name="@nameof(ConfirmProviderViewModel.ProviderId)" value="@Model.ProviderId"/>
+<input type="hidden" id="@nameof(ConfirmProviderViewModel.ProviderName)" name="@nameof(ConfirmProviderViewModel.ProviderName)" value="@Model.ProviderName"/>
```

Comment:
For these inputs is it possible to user the helper Html.HiddenFor(....) for each element.

I think you;re doing this anyway, but EmployerAccountLegalEntityPublicHashedId should be AccountLegalEntityHashedId

ProviderName can be added to the view model or retrieved each time (both ways are valid solutions), I think we need to agree which technique we should standardise on

Reply:
I'm happy to change, however I think we should be consistent in what we are doing on each page, and having a mismatch of razor and tag helper code will only further obfuscate the intention from a front end dev who may not know. We aren't using the Html.BeginForm or a lot of the other ones, so if we are to change this one then we should probably make it uniform for all elements on all views.

Reply:
As agreed with Chris these have been changed to 
```
<input type="hidden" asp-for="FieldName">
```